### ISSUE TITLE
Issue with account

### ISSUE THREAD

**Issue by CTHD98** [VLorin/BotSolebox#1]
<img width="939" alt="Screenshot 2020-03-25 at 14 30 09" src="https://user-images.githubusercontent.com/59808605/77547548-3f5dea00-6ea5-11ea-8716-bce0663600dd.png">
as you can see it won't offer me the choice to either log in or create an account, i have changed my details and put a live account on with an item in the basket, but i can't chose which one to use

**Comment by VLorin** [comment]
Do you have beautifulSoup4 installed on your computer ? 
Btw for choosing between "create an account" or "login" its just by placing comment '#' in main_bot.py (yes that garbage code).
Another point, this project is not very well documented (as you may have notice), so it is a bit tricky to add an item in the basket, you would need to go to the website create the order manually and record the "Network activity"  to copy the data, I do it with chrome.
Last point, right now it looks like the ajoute_chaussure() (wich stand for add_shoes() ) doesn't work anymore, I tried to fix it for you but it still doesn't work, i'm getting a "403 Bad request" reply instead of the "302 Found" when i'm doing the POST request

**Comment by CTHD98** [comment]
yeh i do have beautifulSoup4 :)
Im trying to get this working before tomorrow, it's more to help with checkout with items in my cart already, i have ATC links for getting stuff in the cart, it's just the rest is hard with there 10 min window they offer

**Comment by CTHD98** [comment]
<img width="652" alt="Screenshot 2020-03-25 at 17 16 36" src="https://user-images.githubusercontent.com/59808605/77565614-64aa2280-6ebc-11ea-82d2-92c4af62f0b0.png">
so just put a # infront of the one you don't want like i have done attached?

**Comment by CTHD98** [comment]
i just get an error with "new_session()" is what my problem is :(

**Comment by VLorin** [comment]
Yeah that right, the way you did it, the bot will just log in the website.
About the "new_session()" error, that one is really strange, sometime I get this error too, but when I come back  30 min later to test it again (without changing anything), it works again ¯\_(ツ)_/¯

**Comment by VLorin** [comment]
Just tested it right now, worked fine
![image](https://user-images.githubusercontent.com/49037677/77574712-b8276b00-6ed2-11ea-8656-d4bfa5e3b0bc.png)
